src/Calculation.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Calculation {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        int[] allNumbers = {};
+        int number = 0, pos = 0;
```

Comment:
bitte nicht mehrere variablen in einer zeile deklarieren, auch wenn ihr typ der gleiche ist. das ist nicht verbreitet und nicht besonders leserlich